app/controllers/services_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ServicesController < ApplicationController
+  def register
+    @service = Service.create
+    render json: @service, include: :models
```

Comment:
(@dfaulken's comment I assume got tagged onto the entire issue instead of this comment chain)